### ISSUE TITLE
Missing RootQuery import in generated objects

### ISSUE THREAD

**Issue by Munksgaard** [dillonkearns/elm-graphql#78]
On elm-graphql@1.1.0 I'm getting errors about missing RootQuery types in my generated object files.

Here is my output: 

```
$ elm make src/Project/Object/AuthenticatePayload.elm --output\=elm.js --debug
Detected errors in 1 module.                                         
-- NAMING ERROR ------------------ src/Project/Object/AuthenticatePayload.elm

I cannot find a `RootQuery` type:

42| query : SelectionSet decodesTo RootQuery -> Field (Maybe decodesTo) Project.Object.AuthenticatePayload
                                   ^^^^^^^^^
These names seem close though:

    Order
    Bool
    Never
    Result

Hint: Read <https://elm-lang.org/0.19.0/imports> to see how `import`
declarations work in Elm.
```

The workaround is to insert the following import:

```
import Graphql.Operation exposing (RootQuery)
```

But I'd really want to avoid doing that for all of my files. Besides, I'd prefer not to have to check the generated files into my git repository.

**Comment by Munksgaard** [comment]
It seems like this only happens for mutations...

**Comment by dillonkearns** [comment]
This is fixed in NPM version 1.0.5 ([see the changelog entry](https://github.com/dillonkearns/elm-graphql/blob/master/CHANGELOG-NPM-PACKAGE.md#105---2018-10-31)). Thank you for reporting the issue!
